src/lib/Libutil/hook.c

```diff
@@ -208,6 +210,13 @@ hook_event_as_string(unsigned int event)
 		ev_ct++;
 	}
 
+	if (event & HOOK_EVENT_RESV_END) {
+	    if (ev_ct > 0)
+	        strcat(eventstr, ",");
+	    strcat(eventstr, HOOKSTR_RESV_END);
```

Comment:
You're extending what is being added into this buffer.  Is there a possibility of overflow?

Reply:
The HOOK_BUF_SIZE is defined as 512. By combining all the hook event strings along with comma it comes to 210. so I think so it won't cause an overflow.